src/eslint-config-adeira/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Unreleased
 
 
+# 0.4.0
+
+- Lint all files when `.eslintignore` changes
```

Comment:
We shouldn't write a changelog for versions before 1.0 - they can all be breaking by design and the scanner ignores most of the checks for such versions.